sherpa-onnx/csrc/session.cc

```diff
@@ -21,6 +22,16 @@
 
 namespace sherpa_onnx {
 
+
+static void OrtStatusFailure(OrtStatus *status, std::ostringstream *os) {
```

Comment:
```suggestion
static void OrtStatusFailure(OrtStatus *status, const char *s) {
```

You can pass `os->str().c_str()` to this function